src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -114,6 +114,15 @@ export default class AirportModel {
          */
         this.wip = null;
 
+        /**
+         * This is what AIRAC cycle that the airport has been based off
```

Comment:
This is a nitpick, but I think this could be worded better.  Just not sure how.

@erikquinn @Fechulo @indianbhaji thoughts?

Reply:
I updated the description